pplns-with-job-declaration.tex

```diff
@@ -21,174 +21,193 @@
 
-To remove this incentive, slices must be shorter than the entire mining round. Intuitively, they must be short enough to be possible for the fees to be approximated as constant. Since a slice ends when starts the following one, it is enough define when a slice begins. Note that the slices are created by the pool and recall that they are used to calculate the score based on fees. To each slice is associated a reference job. Fix $f$ the fees of a particular reference job. Let $\delta>0$ chosen by the pool. Then, every share that belong to the slices relative to the reference job must have fees lower than
-\[ f+\delta.\]
-A new slice is created if one of the two event happen:
+To remove this incentive, slices must be shorter than the entire mining round. Intuitively, they must be short enough to be possible for the fees to be approximated as constant. Since a slice ends when a new slice begins, it is enough to define when a slice begins. The slices are created by the pool and the score is calculated based on fees. Each slice has a \emph{reference job}. We assign $f$ the fees of a particular reference job. Let $\delta>0$ be chosen by the pool. Then, every share that belongs to the slice for a reference job must have fees lower than \[ f+\delta.\]
+
+A new slice is created and the reference job for the slice is updated when the pool receives a share in which:
 \begin{enumerate}
-	\item the pool receive a share for a job that have fees bigger or equal then $f+\delta$, where $f$ the actual slice reference job fee. The new slice reference job is this job.
-	\item the pool receive a share for a job that have a prevhash different from the actual slice reference job. The new slice reference job is this job.
+	\item fees are greater than or equal to $f+\delta$ (increased MMEF)
+	\item have a prevhash different from the actual slice reference job (new block)
 \end{enumerate}
-Every share received by the pool that do not have a job that meet condition 1. or 2. is put in the actual slice. The above means that we create a new slice every time that the MMEF change: 1. is when MMEF grows and 2. is when a new block have been found and MMEF is lower or equal.
+Every share received by the pool that does not meet condition 1 or 2 is put in the existing slice. 
+%I'm not certain I articulated this correctly.%
 
 If there are $t$ slices in the window and $F$ are the collected fees of the mined block, assign to each slice $S$ an amount $F_S = F/t$. Then, all the shares will be scored basing on the fees of selected transactions and $F_S$ will be redistributed to all the shares of $S$ accordingly with its fee score.
+% I wonder if this scoring paragraph should be moved below. %
```

Comment:
we are just reserving a part of block reward to every slice, no scoring involved